tests/test_flow/test_reload.py

```diff
@@ -40,216 +40,240 @@ def _write_module(name, contents):
     sys.dont_write_bytecode = dont_write_bytecode
 
 
+@pytest.fixture(params=["reload", "reloading"])
+def reload_method(request):
```

Comment:
nit: I know we approved this before, but can you rename this to something like `reload_type`? It reads like we are passing around a method function.

Reply:
I have modified the `FlowTester` class to actually call the named method i.e. `getattr(.., reload_method)()`. This ensures that there is no typo.

Reply:
How about `reload_method_name`?